packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -324,8 +339,8 @@ const _BaseLink: React.ForwardRefRenderFunction<BladeElementRef, BaseLinkProps>
 
   return (
     <StyledBaseLink
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      ref={ref as any}
+      ref={ref as never}
+      // TODO Check if this is overriden
```

Comment:
Didn't get this todo

Reply:
Will fix this TODO. basically our trigger props are overriding this synthetic events for active state of link

Reply:
Fixed.